### ISSUE TITLE
LGTM standalone | $$config$$config.async method is undefined

### ISSUE THREAD

**Issue by LTe** [square/lgtm#10]
I do not know why but lgtm-standalone.js does not work at all.

```javascript
LGTM.validator().build().validate()
TypeError: $$config$$config.async is not a function
$$promise$$Promise.prototype.then()lgtm-st...?body=1 (wiersz 1567)
$$lgtm$object_validator$$ObjectValidator.prototype.validate()lgtm-st...?body=1 (line 198)
$$config$$config.async(function(){
```

**Comment by eventualbuddha** [comment]
Sorry it took so long to get to this. It is fixed as of v1.0.2.

**Comment by LTe** [comment]
@eventualbuddha thanks a lot!:) :heart:
